cluster/terraform_kubernetes/config/prometheus/development.prometheus.yml

```diff
@@ -101,6 +101,9 @@ scrape_configs:
       target_label: kubernetes_pod_name
     - source_labels: [__meta_kubernetes_pod_node_name]
       target_label: nodename
+    - source_labels: [__meta_kubernetes_namespace]
+      target_label: exported_namespace
+      action: replace
```

Comment:
we already have a target for __meta_kubernetes_namespace as kubernetes_namespace.
Couldn't we just change exported_namespace to kubernetes_namespace in the nginx-ingress-controller.json, then we wouldn't need to change the prometheus.yml?

Reply:
Fixed this and deployed on dev and test